scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -952,6 +955,7 @@ def _response_values_validation(self, expected_response, received_response, resu
                 result.success(check_type, error_success.format(
                     name=expected_name, value=expected_value))
             else:
+                logging.warning(f"Response values validation mismatch: Expected Result: {expected_value} / Received Value: {received_value}")
```

Comment:
So this is the one place where we are not logging the actual expected value.  How about we just fix `error_failure` to log that instead of adding an out-of-band log?

So for example, in this function we could do something like:
```
        error_failure = 'The test expectation "{name} ({actual}) == {value}" is false'
```
and
```
                result.error(check_type, error_failure.format(
                    name=expected_name, actual=received_value, value=expected_value))
```

Reply:
Hi @bzbarsky-apple thanks for your all feedback. Could you review now after the code changes?